yvalenta/Example-socket/simple-server.rb

```diff
@@ -0,0 +1,12 @@
+require 'socket'               # Get sockets from stdlib
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.